lib/OpenQA/BuildResults.pm

```diff
@@ -149,9 +149,8 @@ sub compute_build_results {
         $jr{escaped_id} = $b;
         $jr{escaped_id} =~ s/\W/_/g;
         $jr{reviewed_all_passed} = $jr{passed} == $jr{total};
-        my $failed = $jr{failed} + $jr{softfailed};
-        $jr{reviewed} = $failed > 0 && $jr{labeled} == $failed;
-        $builds{$b} = \%jr;
+        $jr{reviewed}            = $jr{failed} > 0 && $jr{labeled} == $jr{failed};
```

Comment:
yes, can we change that to `$jr{labeled} == $jr{failed}` or do you see problems? Should I prepare that?